app-containers/kubectl/spec

```diff
@@ -0,0 +1,4 @@
+VER=1.30.3
+CHKSUMS="sha512::303c823f74ccc5c1685ec16ba3de9dbbe9614ddf19e279e43ee392a4ef04ee68400a3e0c2a839d6a3f43712fab426084037ed871e3e4caafe8b37999fd83b73a"
+SRCS="https://github.com/kubernetes/kubernetes/archive/v$VER.tar.gz"
```

Comment:
Per the Styling Manual, you should not be using platform-generated tarballs. Use Git tags instead:

```
SRCS="git::commit=tags/v$VER::https://github.com/kubernetes/kubernetes"
```

Also, please move CHKSUMS after SRCS.